core-library/src/main/java/org/silverpeas/core/admin/domain/driver/ldapdriver/LDAPTimeStampMSAD.java

```diff
@@ -49,6 +50,12 @@ else if (lTimeStamp < ((LDAPTimeStampMSAD) other).lTimeStamp)
       return 0;
   }
 
+  @Override
+  public int hashCode() {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) This class overrides "hashCode()" and should therefore also override "equals()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1206)